.github/workflows/ci.yaml

```diff
@@ -17,12 +17,15 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: 
-          - ubuntu-latest
-         #- macOS-latest 
-         #python-version: 
-         #  - 3.6
-         #  - 3.7
+        include:
+          - name: prod
+            os: ubuntu-latest
+            envfile: 'devtools/prod-envs/openff-benchmark-optimization.yaml'
+            envname: 'openff-benchmark-optimization'
```

Comment:
I don't follow the purpose of this test -- Will this just tell us if a new dep is required? Or or a dependency had an API-breaking change?

Reply:
Ah, apologies, I didn't explain this. This is my first attempt to keep the tests around for the existing prod environment, knowing full well it will fail. Perhaps this isn't the best idea, but *eventually* we will advance that environment to match closer to current state.

Reply:
I'm working to get the "latest" env working, though of course many of our dependencies clash. Tracking them down. We'll want to work some of this out anyway.